analyzers/pattern_detector.py

```
@@ -6,293 +6,330 @@
-    
+
     async def _apply_mutation(self, recommendation: Dict) -> Dict:
         """Apply a specific mutation based on recommendation"""
-        protocol = recommendation['protocol']
-        mutation_type = recommendation['mutation_type']
-        
+        protocol = recommendation["protocol"]
+        mutation_type = recommendation["mutation_type"]
+
         # Load current protocol code
         from protocols.loader import load_protocol
```

Ruff (0.11.9)

`protocols.loader.load_protocol` imported but unused

Remove unused import: `protocols.loader.load_protocol`

(F401)